<ui version="4.0" >
 <class>spPrinterDialog</class>
 <widget class="QDialog" name="spPrinterDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Printer file setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Print file name pattern</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="FileNamePattern" >
     <property name="text" >
      <string>*</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="DirSelect" >
     <property name="text" >
      <string>Select print directory</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLabel" name="DirPath" >
     <property name="text" >
      <string>~/tmp</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Area selection</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="AreaSelect" >
     <item>
      <property name="text" >
       <string>Entire data area</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Visible data area</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Output format</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QComboBox" name="FormatSelect" >
     <item>
      <property name="text" >
       <string>.cgm (Computer Graphics Metafile)</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>.ps (PostScript)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="SaveDefault" >
       <property name="text" >
        <string>Save as default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Print1" >
       <property name="text" >
        <string>Print 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PrintAll" >
       <property name="text" >
        <string>Print all</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
